components/proxy/conf/Caddyfile

```diff
@@ -216,7 +216,11 @@ https://*.*.{$GITPOD_DOMAIN} {
 
 	@workspace_port header_regexp host Host ^(webview-|browser-|extensions-)?(?P<workspacePort>[0-9]{2,5})-(?P<workspaceID>[a-z0-9][0-9a-z\-]+).ws(?P<location>-[a-z0-9]+)?.{$GITPOD_DOMAIN}
 	handle @workspace_port {
-		reverse_proxy ws-{re.host.workspaceID}-ports.{$KUBE_NAMESPACE}.{$KUBE_DOMAIN}:{re.host.workspacePort} {
```

Comment:
why are you removing the mapping for workspace ports?

Reply:
ws-proxy does it after adjusting headers: https://github.com/gitpod-io/gitpod/pull/4419/files#diff-39f2e37e657f269523bb53adff2f549f05b1af3db45432b89daa1c2068847ef3R269

Reply:
As far as i understood without this change it does not go through ws-proxy at least in prev env?

Reply:
Correct.

Just in case, I copied this configuration from what we had in NGINX previously.

Reply:
I think in nginx we also delegated to ws-proxy: https://github.com/gitpod-io/gitpod/blob/a0a5017b588f307418b1daf160d8ccee1afbd4b5/chart/config/proxy/vhost.server.conf#L160-L192 but moved it to caddy while switching to nginx on purpose. I am not sure.